cmake/Log_dependencies.cmake

```diff
@@ -16,9 +16,9 @@ FetchContent_Declare( catch-adapter
     GIT_SHALLOW     TRUE
     )
 
-FetchContent_Declare( spdlog-adapter
-    GIT_REPOSITORY  http://github.com/njoy/spdlog-adapter
-    GIT_TAG         origin/build/fetchcontent
+FetchContent_Declare( spdlog
+    GIT_REPOSITORY  https://github.com/gabime/spdlog.git
+    GIT_TAG         a51b4856377a71f81b6d74b9af459305c4c644f8
     GIT_SHALLOW     TRUE
```

Comment:
This will get fixed later, but FYI, you don't need `GIT_SHALLOW` when `GIT_TAG` points to a hash or tag.  It only has meaning for branches.

Reply:
I fixed it and pushed.  I'll have to debate as to whether the commands make sense where I put them or if I should have put them in the main CMakeLists.txt file.  But this will work for now.

Reply:
You beat me to it. I was about to do the same thing.